Docs/user_guide/coordinate_systems.md

```diff
@@ -0,0 +1,268 @@
+origin. This result is also confirmed by the first equations.
+
+The solution of the other equations leads to the following **IJtoLS**-matrix:
+
+$$
+IJtoLS = \begin{pmatrix} 50 & 0 & 50 \\ 0 & -50 & 300 \\ 0 & 0 & 1 \end{pmatrix}
+$$
+
+In the event that a R(A)S basis was used, just the left and anterior axis of
+the anatomical space are flipped, and the image coordinate system appears in
+the same way as in the L\(P\)S case.
+
+![IJtoRS](https://github.com/Slicer/Slicer/releases/download/docs-resources/IJtoRS.png){.bg-warning w=550px align=center}
+
+For this 2D example the **IJtoRS**-matrix would be:
+
+$$
+IJtoRS = \begin{pmatrix} -50 & 0 & 250 \\ 0 & -50 & 300 \\ 0 & 0 & 1 \end{pmatrix}
+$$
+
+This matrix looks very similar to the **IJtoLS**-matrix with 2 differences:
+
+- The translation $\vec{t}$ has changed because of another origin.
+- The right axis is flipped, so the first column of the **IJtoRS**-matrix
+  has just an inverted sign.
+
+## Coordinate system convention in Slicer
+
+DICOM and most medical imaging software use the **LPS coordinate system** for
+storing all data. The choice of origin is arbitrary because only relative
+differences have meaning, so there is no universal standard, but it is often
+set to some geometric center of the imaging system, or it is chosen to be
+near the center of an object of interest.
+
+Both LPS and RAS were in wide use in the early 2000s when development of
+Slicer was started and Slicer developers chose to use the RAS coordinate
+system. Historically scans by GE equipment used RAS while Siemens and others
+used LPS. Since several GE researchers were early contributors to Slicer, RAS
+was adopted for the internal representation.
+
+Slicer still **uses RAS coordinate system for storing coordinate values
+internally** for all data types, but for compatibility with other software, it
+**assumes that all data in files are stored in LPS coordinate system** (unless
+the coordinate system in the file is explicitly stated to be RAS). To achieve
+this, whenever Slicer reads or writes a file, it may need to flip the sign of
+the first two coordinate axes to convert the data to RAS coordinate system.
+
+### Relations to other software/conventions
+
+#### ITK
+
+[ITK](https://itk.org/) uses the LPS convention.
+
+#### Using MATLAB to map Slicer RAS coordinates (e.g. fiducials) to voxel space of a NIfTI Image
+
+To extract the "voxel to world" transformation matrix from a NIFTI file's
+header (entry: `qto_xyz:1-4`) in MATLAB:
+
+```Matlab
+d = inv(M) * [ R A S 1 ]'
+```
+
+where `M` is the matrix and `R A S` are coordinates in Slicer, then `d` gives
+a vector of voxel coordinates.
+
+(Solution courtesy of András Jakab, University of Debrecen)
+
+## References
+
+- http://people.cs.uchicago.edu/~glk/unlinked/nrrd-iomf.pdf
+- http://www.grahamwideman.com/gw/brain/orientation/orientterms.htm
+- http://nifti.nimh.nih.gov/nifti-1/documentation/faq
+- http://teem.sourceforge.net/nrrd/format.html
```

Comment:
```suggestion
- https://teem.sourceforge.net/nrrd/format.html
```

Reply:
Done in [0b7a79e](https://github.com/Slicer/Slicer/commit/0b7a79ed1b791335814ae637ee267ba8edf3750b).